sdk/core/azure-core/azure/core/pipeline/transport/requests_asyncio.py

```diff
@@ -78,6 +78,9 @@ class AsyncioRequestsTransport(RequestsTransport, AsyncHttpTransport):  # type:
     async def __aexit__(self, *exc_details):  # pylint: disable=arguments-differ
         return super(AsyncioRequestsTransport, self).__exit__()
 
+    async def sleep(self, duration):
```

Comment:
Why is this override & call super needed?

Reply:
@annatisch was adding it to her transport PR, there's a problem in azure-core where the async transport was calling a synchronous sleep which resulted in an error because None can't be awaited. Anna thinks it's because the AsyncioRequestsTransport was calling its two base classes in the wrong order.

Reply:
That PR merged, so this should rebase out.